docs/README.md

```diff
@@ -92,6 +93,20 @@ firstLetters('iZir&qym sMdUk* drIXw AmIk* ]', false) // => Zsda
 ```js
 firstLetters('sbid mrY. so mir rhY; iPir. mrY n, dUjI vwr ]', true, true) // => sm.smr;P.mn,dv
 ```
+### isGurmukhi(text, [exhaustive]) ⇒ <code>boolean</code>
+Checks if first char in string is part of the Gurmukhi Unicode block.
+
+**Returns**: <code>boolean</code> - True if Unicode Gurmukhi, false if other.  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| text | <code>String</code> | The text to check. |
+| [exhaustive] | <code>boolean</code> | If `true`, checks if the whole string is Unicode Gurmukhi. |
+
+**Example**  
+```js
+isGurmukhi('ਗੁਰਮੁਖੀ') // => true
isGurmukhi('gurmuKI') // => false
```

Comment:
Formatting output seems
To be wrong here. Check the other funcs to see how to format examples. Please also add an example to examples.js too